web/styles/app_components.css

```diff
@@ -1089,3 +1089,11 @@ div.overlay {
         }
     }
 }
+
+.change_visibility_policy {
+    display: flex;
+
+    i {
+        font-size: 17px;
+    }
+}
```

Comment:
Hmm, I think this belongs in `message_row.css` or something that contains the recipient elements?

Reply:
Moved it to `zulip.css`. 

`.recipient_bar_controls`, `.recipient_row_date` etc lies there.

Reply:
OK. We may move them out but they can go together, so this works.